PokeOneWeb.Data.Tests/Extensions/NatureExtensionsTests.cs

```diff
@@ -0,0 +1,228 @@
+﻿using FluentAssertions;
+using PokeOneWeb.Data.Entities;
+using PokeOneWeb.Data.Extensions;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace PokeOneWeb.Data.Tests.Extensions
+{
+    public class NatureExtensionsTests
+    {
+        [Fact]
+        public void GetDescription_PositiveAttackValue_CorrectString()
+        {
+            // Given
+            Nature nature = new Nature()
+            {
+                Attack = 1
+            };
+
+            // When
+            string description = nature.GetDescription();
+
+            // Then
+            description.Should().Be("+Atk", "because only the attack value was positive.");
```

Comment:
Every nature increases one stat and decreases one stat. Imo it would make more sense to only verify "valid" combinations, instead of single stats, or all six stats (Either some examples or all 25 possible natures). Additionally, negative test cases could be added, where nature.GetDescription() throws an Exception, when a nature is called which does not follow the rule of "boost 1 stat, reduce 1 stat".